ci/scripts/integration_spark.sh

```diff
@@ -73,14 +73,38 @@ pushd ${spark_dir}
 
   # Run pyarrow related Python tests only
   spark_python_tests=(
-    "pyspark.sql.tests.test_arrow"
-    "pyspark.sql.tests.test_pandas_map"
-    "pyspark.sql.tests.test_pandas_cogrouped_map"
-    "pyspark.sql.tests.test_pandas_grouped_map"
-    "pyspark.sql.tests.test_pandas_udf"
-    "pyspark.sql.tests.test_pandas_udf_scalar"
-    "pyspark.sql.tests.test_pandas_udf_grouped_agg"
-    "pyspark.sql.tests.test_pandas_udf_window")
+    "pyspark.sql.tests.test_arrow")
+
+  old_test_modules=false
+  # Spark <= 2.x or <= 3.3.x
+  if [ "${SPARK_VERSION:0:2}" == "2." ]; then
+    old_test_modules=true
+  elif [ "${SPARK_VERSION:0:2}" == "3." ]; then
+    if [ "${SPARK_VERSION:2:2}" == "1." ] || \
+       [ "${SPARK_VERSION:2:2}" == "2." ] || \
+       [ "${SPARK_VERSION:2:2}" == "3." ]; then
+        old_test_modules=true
+    fi
+  fi
```

Comment:
FYI: We may be able to use `case` for this:

```suggestion
  case "${SPARK_VERSION}" in
    1.*|2.*|3.0.*|3.1.*|3.2.*|3.3.*)
      old_test_modules=true
      ;;
    *)
      old_test_modules=false
      ;;
  esac
```

Reply:
Beautiful. Maybe `v1.*|v2.*|v3.0.*|v3.1.*|v3.2.*|v3.3.*` could work in this case.

Reply:
Thanks @kou ! Yes, this is much better than what I was doing!